assembl/views/api/auth.py

```diff
@@ -307,3 +311,16 @@ def get_user_has_permission(request):
     if not discussion:
         raise HTTPNotFound("Discussion %d does not exist" % (discussion_id,))
     return a_users_with_permission(discussion_id, permission)
+
+
+@view_config(route_name='saml_metadata')
+def saml_metadata_view(request):
+    complete_url = request.route_url('social.complete', backend="saml")
+    saml_backend = load_backend(
+        load_strategy(request),
```

Comment:
Nvm, now I understand what a `Backend` is to python social auth, and what the `Strategy` object does...read [here](https://github.com/omab/python-social-auth/tree/master/social/backends)